src/protocol.cpp

```diff
@@ -619,12 +619,11 @@ void CProtocol::CreateAndImmSendConLessMessage ( const int iID, const CVector<ui
 
 void CProtocol::ParseMessageBody ( const CVector<uint8_t>& vecbyMesBodyData, const int iRecCounter, const int iRecID )
 {
-    // clang-format off
-/*
-// TEST channel implementation: randomly delete protocol messages (50 % loss)
-if ( rand() < ( RAND_MAX / 2 ) ) return false;
-*/
-    // clang-format on
+    // TEST: BEGIN
+    /* channel implementation: randomly delete protocol messages (50 % loss)
+     */
+    // if ( rand() < ( RAND_MAX / 2 ) ) return false;
+    // TEST: END
```

Comment:
Honestly, these tests should not be in the code but somewhere externally. But that's out of scope of this PR